pkgs/development/ocaml-modules/sqlexpr/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildOcaml, fetchurl, ocaml, findlib, ocaml_batteries, csv, ocaml_lwt, ocaml_sqlite3, estring }:
+
+buildOcaml rec {
+  name = "sqlexpr";
+  version = "0.5.5";
+
+  src = fetchurl {
+    url = "https://forge.ocamlcore.org/frs/download.php/1203/ocaml-sqlexpr-${version}.tar.gz";
+    sha256 = "02pi0xxr3xzalwpvcaq96k57wz2vxj20l2mga1a4d2ddvhran8kr";
+  };
+
+  buildInputs = [ ocaml_batteries ocaml_lwt ];
+
+  propagatedBuildInputs = [ csv estring ocaml_sqlite3 ];
+
```

Comment:
According to the `META` file, batteries and lwt should be propagated as well.

```
requires = "csv batteries sqlite3 estring lwt lwt.syntax lwt.unix unix threads"
```

And as for `estring`, the `ocaml` and `findlib` arguments are never used.